terms-of-use.md

```diff
@@ -0,0 +1,336 @@
+# Status Terms of Use
+
+Last updated: 19 September 2024
+
+### 1. Introduction
+&nbsp;
```

Comment:
i'm not sure if `&nbsp;` is needed

Reply:
Probably not, I think it would be fine to remove all `&nbsp`s. As long as the files are identical in all repos then it will be fine.